src/emscripten-optimizer/colors.h

```diff
@@ -17,6 +17,7 @@
 #ifndef wasm_color_h
 #define wasm_color_h
 
+#ifndef WIN32
 #include <unistd.h>
```

Comment:
Can't you just conditionally include unistd and the #endif right after? The rest of the file could stay the same IIUC.